cmd/root.go

```diff
@@ -21,22 +21,20 @@ var reboot bool
 var container string
 var exit_77 bool
 
-// the number of times to retry commands that pull data from the network
-const numRetriesNetCommands = 5
+const (
+	// the number of times to retry commands that pull data from the network
+	numRetriesNetCommands = 5
+	etcPivotFile          = "/etc/pivot/image-pullspec"
+	runPivotRebootFile    = "/run/pivot/reboot-needed"
```

Comment:
Nothing seems to create this?

Reply:
This would be created from Ignition if one wants first boot pivoting.

Reply:
OK right, sorry I think I remember now.  This is for the "pivot before cluster" path.